wallet/api/openrpc.json

```diff
@@ -5,6 +5,82 @@
     "title": "Wallet API"
   },
   "methods": [
+    {
+      "name": "client.check_transaction",
+      "summary": "Check a transaction without sending it.",
+      "description": "This method checks a transaction or a bundle of transactions and returns them to the third-party application, without sending it to the network. What happens with the transaction is up to the third-party application.\n\nThe user has to review the transaction.",
```

Comment:
Check _against the network_. That bit is important so user knows the transaction is compared with the inner state of the network

Reply:
Addressed here https://github.com/vegaprotocol/vega/pull/7657/commits/061417ef72aebed33d0cbbc49c5b46f32310fe14#diff-4541166dc7cdf695bbc57461645d25da8509eeb5839947816d4979a654eeb880R10-R11 as well as the above comment
Thanks!